Makefile

```diff
@@ -94,7 +94,8 @@ tests:
 	go test ./...
 
 vet:
-	go vet ./...
+	go tool vet --all .
+	! go tool vet --shadow . 2>&1 | grep -vE 'declaration of err shadows declaration at (proto/text.go:535|proto/text.go:570|protoc-gen-gogo/main.go:84)'
```

Comment:
heh, if it's not in the automated build it may as well be left out entirely. it will not provide any help in preventing these bugs (unless you're planning on running `make vetshadow` yourself on every PR?)